src/lib/update-versions.js

```diff
@@ -17,7 +17,7 @@ const filter = ({ internal, external, dep, internalScopes, ignoreScopes }) => {
 const updateVersions = async (content, saveError, options = {}, depsBar) => {
     const { name: packageName, dependencies = {}, devDependencies = {}, peerDependencies = {} } = content;
     const { major, minor, patch, deps, dev, peer, concurrency, pattern } = options;
-    const matchRegEx = new RegExp(pattern || (options._ && options._[0]) || '', 'i');
+    const matchRegEx = new RegExp(pattern, 'i');
```

Comment:
pattern isn't the only way to add a pattern.  The final argument is also assumed to be the pattern i.e. `yarn sonar webapp` where `webapp` is the pattern

Reply:
Ahh, I was thinking what it was